Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.py

```diff
@@ -95,6 +95,61 @@ def get_user(self, user_id):
         return response
 
 
+def fetch_incidents(client: MsGraphClient, fetch_time: str, fetch_limit: int, filter_param: str, providers_param: str) \
+        -> list:
+    filter_query = ""
+    if providers_param:
+        providers_query = []
+        providers_lst = providers_param.split(',')
+        for provider in providers_lst:
+            providers_query.append(f"vendorInformation/provider eq '{provider}'")
+        filter_query = (" or ".join(providers_query))
+    if filter_param:  # overrides the providers query, if given
+        filter_query = filter_param
+
+    severity_map = {'low': 1, 'medium': 2, 'high': 3, 'unknown': 0, 'informational': 0}
+
+    last_run = demisto.getLastRun()
+    timestamp_format = '%Y-%m-%dT%H:%M:%S.%fZ'
+    now = datetime.now().strftime(timestamp_format)
+    if not last_run:  # if first time running
+        new_last_run = {'time': parse_date_range(fetch_time, date_format=timestamp_format)[0]}
+    else:
+        new_last_run = last_run
+    demisto_incidents: List = list()
+    time_from = new_last_run.get('time')
+    time_to = datetime.now().strftime(timestamp_format)
+
+    # Get incidents from MS Graph Security
+    demisto.debug(f'Fetching MS Graph Security incidents. From: {time_from}. To: {time_to}\n')
+    incidents = client.search_alerts(last_modified=None, severity=None, category=None, vendor=None, time_from=time_from,
+                                     time_to=time_to, filter_query=filter_query)['value']
+
+    if incidents:
+        count = 0
+        incidents = sorted(incidents, key=lambda k: k['createdDateTime'])  # sort the incidents by time-increasing order
+        last_incident_time = last_run.get('time', '0')
+        demisto.debug(f'Incidents times: {[incidents[i]["createdDateTime"] for i in range(len(incidents))]}\n')
+        for incident in incidents:
+            incident_time = incident.get('createdDateTime')
+            if incident_time > last_incident_time and count < fetch_limit:
+                demisto_incidents.append({
+                    'name': incident.get('title'),
```

Comment:
is title unique to every alert? if so good, if not wi should consider adding an id of the alert to it as the incident name.
also we should consider adding the integration name. lets talk about this.

Reply:
The title isn't unique, but there is an unique ID for every alert. I updated the name of the incident to be 'title + id', for example: 'test alert - da637218501473413212_-1554891308'.
What do you mean by adding the integration name?